pkg/operator/discover/discover.go

```diff
@@ -190,18 +194,76 @@ func (d *Discover) createDiscoverDaemonSet(namespace, discoverImage string) erro
 func ListDevices(context *clusterd.Context, namespace, nodeName string) (map[string][]sys.LocalDisk, error) {
 	var devices map[string][]sys.LocalDisk
 	listOpts := metav1.ListOptions{LabelSelector: fmt.Sprintf("%s=%s", k8sutil.AppAttr, discoverDaemon.AppName)}
+	// wait for device discovery configmaps
+	retryCount := 0
+	retryMax := 30
+	sleepTime := 5
+	for {
+		retryCount++
```

Comment:
Shouldn't you increase the counter at the end of each iteration?

Reply:
yes, it can be done in that way too, this is mostly mirroring the convention in existing code.